dask/multiprocessing.py

```diff
@@ -190,11 +201,25 @@ def get(
             # https://github.com/dask/dask/issues/6640.
             os.environ["PYTHONHASHSEED"] = "6640"
         context = get_context()
+        initializer = partial(
+            initialize_worker_process,
+            user_initializer=initializer,
+            override=(
+                replace_default_initializer
+                or config.get("multiprocessing.replace_initializer", False)
+            ),
+        )
```

Comment:
Think we can skip modifying this if we don't need to

```suggestion
        if replace_initializer:
            initializer = partial(
                initialize_worker_process,
                user_initializer=initializer,
            )
```